bindings/python/tests/test_legged_odometry.py

```diff
@@ -0,0 +1,93 @@
+import pytest
+pytestmark = pytest.mark.floating_base_estimators
+
+import bipedal_locomotion_framework.bindings as blf
+
+
+def test_legged_odometry():
+    # This function just performs an interface test for
+    # the generated python bindings,
+
+    # create KinDynComputationsDescriptor
+    kindyn_handler = blf.StdParametersHandler()
+    kindyn_handler.set_parameter_string("model_file_name", "./model.urdf")
+    kindyn_handler.set_parameter_vector_string("joints_list", ["neck_pitch", "neck_roll", "neck_yaw",
+                                                               "torso_pitch", "torso_roll", "torso_yaw",
+                                                               "l_shoulder_pitch", "l_shoulder_roll", "l_shoulder_yaw",
+                                                               "l_elbow",
+                                                               "r_shoulder_pitch", "r_shoulder_roll", "r_shoulder_yaw",
+                                                               "r_elbow",
+                                                               "l_hip_pitch", "l_hip_roll", "l_hip_yaw",
+                                                               "l_knee", "l_ankle_pitch", "l_ankle_roll",
+                                                               "r_hip_pitch", "r_hip_roll", "r_hip_yaw",
+                                                               "r_knee", "r_ankle_pitch", "r_ankle_roll"])
+    kindyn_desc = blf.construct_kindyncomputations_descriptor(kindyn_handler)
+    assert(kindyn_desc.is_valid())
```

Comment:
No need for `()` in asserts, just a space

Reply:
Thanks! Fixed in https://github.com/dic-iit/bipedal-locomotion-framework/pull/218/commits/69f96dd14d124f1d152ca304bf8546b1a8066710